### PR TITLE
m6 change terms of service to terms of use, add link and blog of COMPSs

### PR DESCRIPTION
**Pull request by SergioLangaritaBenitez**
-Changed the name of the page "terms of service" to "terms of use".
-Added the link of the IM dashboard privacy policy in /terms-conditions.
-Added a blog of OSCAR with the integration of COMPSs.

### CODE REVIEW

File: content/english/blog/post-guide-to-use-COMPSS-in-OSCAR.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: "Use COMPSs with OSCAR"
+date: 2023-02-02T09:00:00+01:00
+# post image
+image: "https://compss-doc.readthedocs.io/en/stable/_images/COMPSs_logo.png"
```
Comment (gmolto) [inline content/english/blog/post-guide-to-use-COMPSS-in-OSCAR.md:5]: The image is shown too big in the blog section.

**Comment by gmolto** [comment]
Changes introduced, please re-check.

**Comment by SergioLangaritaBenitez** [comment]
The main image on the Compss blog had changed into a new one. The changes introduced are nice.

**Review by gmolto — APPROVED** [review]
LGTM

**Pull request merged**